main.js

```diff
@@ -46,39 +62,57 @@ function createCard(idea) {
               <label class="button-label-edit" for="">Comment</label>
             </div>
           </article>`
-}
+};
 
 function loadCards() {
   for (var i = 0; i < localStorage.length; i++) {
     var key = localStorage.key(i);
     var object = localStorage.getItem(key);
-    var parsedObject = JSON.parse(object);
-    allCards[i] = parsedObject;
+    var card = JSON.parse(object);
+    var instanciatedIdea = new Idea(card);
+    allCards[i] = instanciatedIdea;
+  };
+  orderLoadedCards();
+};
 
-    var cardHTML = createCard(parsedObject);
+function starLoad(idea) {
+  if (idea.starred) {
+    return "assets/star-active.svg";
+  } else {
+    return "assets/star.svg";
   }
+}
+
+function orderLoadedCards() {
+  allCards.sort(function(a, b) {
+    return a.id - b.id;
```

Comment:
I'd try to go to arrow function callbacks across the board for consistency if you can